tools/codegen/gen_functionalization_type.py

```diff
@@ -29,6 +32,11 @@
 #     Generates view_copy() composite kernels for all view_copy operators.
 
 
+FunctionalizationTarget = Enum('Target', (
```

Comment:
'FunctionalizationTarget', not 'Target'

Reply:
Also, Target is intended to be open, so you could add these there. Though, I'd probably try to follow the existing naming convention (which tries to say if you are DEFINITION or DECLARATION in the name) more closely